Doc/library/sqlite3.rst

```diff
@@ -1111,15 +1136,13 @@ Row Objects
 
 .. class:: Row
 
-   A :class:`Row` instance serves as a highly optimized
+   A :class:`Row` instance that serves as a highly optimized
    :attr:`~Connection.row_factory` for :class:`Connection` objects.
-   It tries to mimic a tuple in most of its features.
-
-   It supports mapping access by column name and index, iteration,
+   It tries to mimic a :class:`tuple` in most of its features,
+   and supports :term:`mapping` access by column name and index, iteration,
    representation, equality testing and :func:`len`.
```

Comment:
```suggestion
   and supports iteration; :func:`repr`, equality testing, :func:`len`,
   and :term:`mapping` access (by column name and index).
```

* Restructure to avoid syntactic ambiguity
* Use (what I presume is the intended) `repr` instead of "representation", which is much more clear about what it refers to (I've rarely if ever heard `repr` called that)

Reply:
That's better, but I prefer to leave out the parentheses around column name and index.